docs/release_notes/release_notes-0.14.0.rst

```diff
@@ -28,10 +29,11 @@ Key Features and Updates
 * Documentation improvements
   * DOCS-#4077: Add release notes template to docs folder (#4078)
   * DOCS-#4082: Add pdf/epub/htmlzip formats for doc builds (#4083)
+  * DOCS-#4079: Fix link to PandasDataframe in docs (#4108)
```

Comment:
```suggestion

```
Was this line added here by mistake?

BTW, the PR that closes the [DOCS-#4079](https://github.com/modin-project/modin/issues/4079) issue is #4080 but not #4108